.github/workflows/build.yaml

```diff
@@ -200,21 +205,23 @@ jobs:
                 MERGE_METHOD: squash
                 MERGE_LABELS: ""
 
-            - name: Check for PR merge
+            - name: Get master branch sha
+              id: master_sha
               if: ${{ steps.check_version_updated.outputs.updated == 'true'}}
               run: |
-                ve1/bin/check-auto-merge --api-url=${{ github.event.pull_request._links.self.href }}
+                git fetch
+                export ORIGIN_MASTER_SHA=$(git rev-parse origin/master)
+                echo "origin_master_sha=$ORIGIN_MASTER_SHA" >> $GITHUB_OUTPUT
 
-            - name: Create the the release
-              id: create_release
+            - name: Create release tag
+              id: create_release_tag
               if: ${{ steps.check_version_updated.outputs.updated == 'true'}}
-              uses: softprops/action-gh-release@v1
+              uses: mathieudutour/github-tag-action@v6.1
               with:
-                tag_name: ${{ steps.check_version_in_PR.outputs.PR_version }}
-                body: ${{ steps.check_version_in_PR.outputs.PR_release_body }}
-                files: ${{ steps.check_version_in_PR.outputs.PR_tarball_name }}
-              env:
-                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                github_token: ${{ secrets.PAT_MGOERENS }}
```

Comment:
```suggestion
                github_token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Sorry I'm a bit too verbose sometimes, I mentioned in a previous comment:

> In the build.yaml workflow, the steps which tags the main branch is run using a Personal Access Token (currently set to PTA_MGOERENS, but that secret doesn't exist in this repo). This is because by default, a GitHub Workflow cannot trigger another GitHub Workflow. See https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow

So I believe we must use a Personal Access Token. We'll have to create the secret in this repo before merging (before we cut a new release). I'm happy to rename it to something else than `PAT_MGOERENS` though.